### DISCUSSION TITLE
[#1116763] Project import from mcwrold completley breaks the Editor

### DISCUSSION THREAD

**Discussion by RASMiranda** [Mojang/minecraft-editor#136] [Bugs]
When I open the world xW5VYXnaBQA=.mcworld, at the begining it's ok, but after a bit the Editor is broken, can only move but doesn't respond to any other action.
Here's the mcworld file: https://1drv.ms/u/s!AhvpvVNhzGsQh_0668GFWHDKCE50rA?e=mUYvHt
Editor Preview version 1.20.20.20
Microsoft Windows 11 Home Version	10.0.22621 Build 22621 x64-based PC

**Comment by dacowan** [comment]
Very cool looking world, but I'm not seeing any issues.  Can you tell me what you were doing when you started to see problems?  Did you see anything in the log window?

**Comment by RASMiranda** [comment]
just aftger a while of using the tool, it's stops responding, here's a video:
https://1drv.ms/v/s!AhvpvVNhzGsQh_07XMBP92ZHWJn27Q?e=A7Z7jy

**Reply by dacowan** [reply]
Would you mind going to `WIN-Key > About Your PC` from the start menu, and posting the device specifications.
From the movie, it looks like you might be running on a laptop and the server simulation rate is very very low.

**Reply by RASMiranda** [reply]
Device name	MSI
Processor	12th Gen Intel(R) Core(TM) i7-12650H   2.30 GHz
Installed RAM	16.0 GB (15.7 GB usable)
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Edition	Windows 11 Home
Version	22H2
Installed on	‎31/‎12/‎2022
OS build	22621.1992
Experience	Windows Feature Experience Pack 1000.22644.1000.0
